examples/full-example/src/email-password/signup.ts

```diff
@@ -82,6 +82,7 @@ async function createGraphcoolUser(api: GraphQLClient, email: string, password:
   const variables = {
     email,
     password: password,
+    name: name,
```

Comment:
Only this one got left behind.

Reply:
@kbrandwijk sorry :(
I just changed it.